game-core/src/main/java/games/strategy/engine/message/RemoteActionCode.java

```diff
@@ -0,0 +1,19 @@
+package games.strategy.engine.message;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation that sets the "opcode" for any particular method that may be used over the network
```

Comment:
I'd perhaps rephrase this slightly.
- opcode does not have a well defined meaning, perhaps better to say something along the lines of: "Annotation that defines a method number, methods are called by method number using the {@link RemoteMethodCall} mechanism}". In order to stay compatible over the network, annotated methods should be altered (including: renaming, changes in signature, or changes in the method ordering number)."